Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonLinkWrapLabel.cs

```diff
@@ -527,7 +527,10 @@ private void OnPaletteChanged(EventArgs e)
             // Layout and repaint with new settings
             NeedPaint(true);
 
-            PaletteChanged.Invoke(this, e);
+            if (PaletteChanged != null)
+            {
+                PaletteChanged.Invoke(this, e);
```

Comment:
Use the compressed style, so that it matches the rest of the invokes

Reply:
Now uses `?`